ecs-cli/modules/cli/attributechecker/attribute_checker_app.go

```diff
@@ -0,0 +1,171 @@
+// Copyright 2015-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package attributechecker
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	ecsclient "github.com/aws/amazon-ecs-cli/ecs-cli/modules/clients/aws/ecs"
+	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/flags"
+	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/config"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/docker/libcompose/project"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+const (
+	displayTitle            = true
+	containerInstanceHeader = "Container Instance"
+	missingAttributesHeader = "Missing Attributes"
+)
+
+var infoColumns = []string{containerInstanceHeader, missingAttributesHeader}
+
+//AttributeChecker will compare task def and containers instances attributes and outputs missing attributes
+func AttributeChecker(c *cli.Context) {
+	err := validateAttributeCheckerFlags(c)
+	if err != nil {
+		logrus.Fatal("Error executing 'Attribute Checker': ", err)
+	}
+	rdwr, err := config.NewReadWriter()
+	if err != nil {
+		logrus.Fatal("Error executing 'Attribute Checker': ", err)
+	}
+	commandConfig, err := config.NewCommandConfig(c, rdwr)
+	if err != nil {
+		logrus.Fatal("Error executing 'Attribute Checker': ", err)
+	}
+
+	ecsClient := ecsclient.NewECSClient(commandConfig)
+
+	taskDefAttributeNames, err := taskdefattributesCheckRequest(c, ecsClient)
+	if err != nil {
+		logrus.Fatal("Error executing 'Attribute Checker': ", err)
+	}
+	if len(taskDefAttributeNames) == 0 {
+		logrus.Info("The given task definition does not have any attributes")
```

Comment:
As mentioned previously, can you make it so that the command terminates in this if block?

```
if len(taskDefAttributeNames) == 0 {
		logrus.Info("The given task definition does not have any attributes")
                 return
}
```

As far as I can tell, there's no point in running the rest of the code if the task def has no attributes.